model/metrics/vcs.yaml

```diff
@@ -0,0 +1,150 @@
+groups:
+  - id: metric.vcs.repository.change.count
+    type: metric
+    metric_name: vcs.repository.change.count
+    brief: 'The number of changes (pull requests) in a repository, categorized by their state (either open or merged)'
+    instrument: gauge
+    unit: "{change}"
+    stability: experimental
+    #extends: metric_attributes.vcs
+    attributes:
+      - ref: vcs.change.state
+        requirement_level: required
+      - ref: vcs.repository.name
+        requirement_level: required
+  - id: metric.vcs.repository.change.time_open
+    type: metric
+    metric_name: vcs.repository.change.time_open
```

Comment:
suggesting `vcs.repository.change.time_to_close` or `vcs.repository.change.request.duration`

It might be interesting to look into CHAOSS metrics and see what would apply https://github.com/chaoss/wg-evolution/blob/main/focus-areas/code-development-process-quality/change-request-reviews.md

Reply:
I've stayed away from aggregation metrics purposefully because getting the raw telemetry enables those calculations. Generally speaking, all these metrics come out of [openo11y's leading indicators](https://openo11y.dev/human-systems/delivery-metrics/leading-delivery-indicators/) which are closely aligned, with [dora.dev](https://dora.dev/capabilities/)'s capabilities.

Reply:
I'm resolving this conversation.
If you think this requires further discussion please reopen it or create a follow-up issue.